NEWS

```diff
@@ -2,13 +2,235 @@ systemd System and Service Manager
+          encrypted key from the LUKS2 JSON embedded metadata header. This
+          allows the information how to open the encrypted device to be
+          embedded directly in the device and obviates the need for
+          configuration in an external file.
+
+        * LUKS devices may now be unlocked using TPM2 hardware.
+
+        * systemd-repart may lock partitions using TPM2 hardware. This may be
+          useful for example to create an encrypted /var partition bound to the
+          machine on first boot.
+
+        * A new systemd-cryptenroll tool has been added to enroll FIDO2+PKCS#11
+          security tokens to LUKS volumes, list and destroy them. See
+          https://www.freedesktop.org/software/systemd/man/systemd-cryptenroll.html.
+
+        * The manager may be configured as compile time to use fexecve instead
+          of execve when spawning children. Using fexecve closes a window
+          between checking the security context of an executable and spawning
+          it, but unfortunately the kernel displays stale information in the
+          comm field, which impacts ps output and such.
+
+        * The configuration option -Dcompat-gateway-hostname has been dropped.
+          "_gateway" is now the only supported name.
+
+        * The ConditionSecurity=tpm2 unit file setting may be used to check
+          if the system has at least one TPM2 (tpmrm class) device.
+
+        * The tables of system calls in seccomps filters are now automatically
+          generated from kernel lists exported on
+          https://fedora.juszkiewicz.com.pl/syscalls.html.
+
+          The following architectures should now have complete lists:
+          alpha, arc, arm64, arm, i386, ia64, m68k, mips64n32, mips64, mipso32,
+          powerpc, powerpc64, s390, s390x, tilegx, sparc, x86_64, x32.
+
         * The MountAPIVFS= service file setting now additionally mounts a tmpfs
-          on /run/ if it is not already a mount point. A writable /run/ has always
-          been a requirement for a functioning system, but this was not
+          on /run/ if it is not already a mount point. A writable /run/ has
+          always been a requirement for a functioning system, but this was not
           guaranteed when using a read-only image.
-          Users can always specify BindPaths= or InaccessiblePaths= as overrides,
-          and they will take precedence. If the host's root mount point is used,
-          there is no change in behaviour.
+
+          Users can always specify BindPaths= or InaccessiblePaths= as
+          overrides, and they will take precedence. If the host's root mount
+          point is used, there is no change in behaviour.
+
+        * New bind mounts and file system image mounts may be injected into the
+          mount namespace of a service (without restarting it). This is exposed
+          as 'systemctl mount-image <unit> <image>…'.
```

Comment:
should this also show 'systemctl bind' for the bind mounts case, since it's mentioned?

Reply:
Please just submit a follow-up pr... This will be quicker than handling each typo and omission through comments.

Reply:
sure, will do once this is merged